starter/package.json

```diff
@@ -62,7 +62,6 @@
     "postinstall": "npm run webdriver:update",
     "postversion": "git push && git push --tags",
     "preclean:install": "npm run clean",
-    "precommit": "lint-staged",
```

Comment:
Why is this removed in the starter if `lint-staged` is still in the `devDependencies` list?

Reply:
Because this script is useless.
It is not present in our main package.json neither

Reply:
Ooops... I missed that, sorry :p